components/compass-runtime-agent/internal/kyma/applications/converter_test.go

```diff
@@ -22,8 +22,8 @@ func TestConverter(t *testing.T) {
 			ID:          "App1",
 			Name:        "Appname1",
 			Description: "Description",
-			Labels: map[string][]string{
-				"key": {"value1", "value2"},
+			Labels: map[string]interface{}{
+				"key": []string{"value1", "value2"},
```

Comment:
I think it would be good to add another label of type `string` to hit the second `case` statement in the switch.